confgenerator/testdata/valid/linux/metrics-receiver_elasticsearch/input.yaml

```diff
@@ -0,0 +1,23 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+metrics:
+  receivers:
+    elasticsearch_default:
```

Comment:
Can we remove the `_default` part here?

For the context: Log names are derived from logging receiver IDs, and the dropdown in UI + sample query / dashboard relies on the log name. For the sample configuration, we should just make the receiver IDs match the receiver type. So the sample dashboards and queries can use that. If users customize it, they can then adjust dashboards and queries accordingly.
The same could apply to metrics receivers and pipelines for consistency.

Reply:
Yeah, for sure. I was treating these as tests only, and not as example configurations, but it seems like we're treating them as both here.

Reply:
Thanks!